test/README.md

```diff
@@ -126,3 +126,29 @@ You should see the test pass, checking that message delivery works correctly and
 ## Generating/Updating new test fixtures
 
 Test fixtures are taken by running the service in full e2e test. The relayer should log the fixture data you need (code is in [the relayer here](../relayer/workers/beefyrelayer/fixture-data-logger.go), though may require a bit of manual copy/pasting to get perfectly it in the right format.
+
+## Running E2E tests on Ropsten
+
+To run the E2E tests on Ropsten you need to have separate accounts for the relayers, an account for deployment and one for running the E2E test stack. You will also require an [Infura](https://infura.io/) account and project.
+
+Add the following variables to `ethereum/.envrc`:
+```bash
+export ROPSTEN_PRIVATE_KEY=0x…                # Your deployment account private key
+export INFURA_PROJECT_ID=…                    # Your Infura project id
+```
+
+Add the following variables to `test/.envrc`:
+```bash
```

Comment:
We already have an envrc template at `test/.envrc-example`, where these example vars should go (at least the mandatory ones). The optional vars could perhaps be commented out in the template file.